### ISSUE TITLE
[Beehave] Add another "run simulation" button at the bottom, before the results

### ISSUE THREAD

**Issue by csckata** [BioDT/biodt-shiny#110]
Another button has been requested as one button at the start of the page is not the most logical placement, so another button should be added after all the parameters have been given, but before the results card

**Comment by csckata** [comment]
Hi @osalamon & @Nithador ! I added some of the points Juergen pointed out in our 24/10/2024 meeting so we could maybe discuss how to go forward with these

**Comment by osalamon** [comment]
@csckata Thanks very much for putting all of three Juergen's enhancements here on GitHub;-)

About this issue: maybe I misunderstood Juergen but I think he didn't want to "copy" that button. In my understanding he wanted to move that button exactly to the place as you described (ie. below Lookup table and above Output plot) and remove it from the current position completely.

It makes more sense to me that way - I think it wouldn't be the best practice to have two buttons with exactly the same functionality on one page, it could be somehow confusing for the users, I think.

Of course, I might be wrong and misunderstood what Juergen said...

**Comment by osalamon** [comment]
Okay, mates:-) @Nithador @csckata 

This issue should be solved, you can check it out in its branch: https://github.com/BioDT/biodt-shiny/tree/110-beehave-add-another-run-simulation-button-at-the-bottom-before-the-results

Basically, it wasn't the most hardcore task I've done in my life:-D I just added one another additional module, called `beekeeper_runsimulation.R` besides the module where the button "Run simulation" was placed originally (= `beekeeper_control.R`). I moved all relevant UI as well as server logic.

As always - when somebody is doing programming of something in general -, various hiccups appear. This branch is no exception:-) But nothing tragic, I am glad that it took me not so much time all in all.

**Comment by osalamon** [comment]
Two things I'd like to ask you whenever you will have few minutes of spare time:

1. (This one is more on @csckata probs.) What about the new design of the page? I tried to separate the card with "new" Run simulation visually somehow with the help of bslib's "card". But even me, graphic/design-blind person, it doesn't look OK:
![image](https://github.com/user-attachments/assets/7a8788de-30c0-4cb7-bb32-d12e77e73ca5)

2. (@csckata , @Nithador) Any other feedback is welcomed. Or you can just go to the Honeybee pDT page, make some clicks here and there... and basically confirm me that the functionality wasn't f**ked up by my changes.

Many thanx,
O
